utils/install.py

```diff
@@ -280,20 +280,20 @@ def compare(self, version2, separator=". |-", ignorecase=True):
 }
 
 SUPPORTED_EXTENSIONS_VERSION = {
-    "Linux" + "x86_64" + TENSORFLOW: VersionString("0.9.0"),
+    "Linux" + "x86_64" + TENSORFLOW: VersionString("TF-2.6.0"),
```

Comment:
Maybe this should not be changed?
The version of extensions should be the same as WasmEdge, the `TF-2.6.0` or `TF-2.6.0-CC` should be used only in Tensorflow-dependencies.

Reply:
Yes, I have reverted this, thank you